sound/soc/sof/topology.c

```diff
@@ -269,6 +269,8 @@ static int sof_control_load_volume(struct snd_soc_component *scomp,
 	struct snd_sof_dev *sdev = snd_soc_component_get_drvdata(scomp);
 	struct snd_soc_tplg_mixer_control *mc =
 		(struct snd_soc_tplg_mixer_control *)hdr;
+	struct sof_ipc_ctrl_data *cdata;
+	int i, ret, tlv[TLV_ITEMS];
```

Comment:
can you use one line per declaration, it's more readable and consistent with the rest of the code.